org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/ChangeConverter.xtend

```diff
@@ -88,12 +106,24 @@ class ChangeConverter implements IAcceptor<IEmfResourceChange> {
 					val range = new Range(start, end)
 					new TextEdit(range, replacement.replacementText)
 				]
-				addTextEdit(uri, textEdits)
+				addTextEdit(uri, document, textEdits)
 			]
 		}
 	}
 	
-	protected def addTextEdit(URI uri, TextEdit... textEdit) {
-		edit.changes.put(uri.toString, textEdit)
+	protected def addTextEdit(URI theUri, Document document, TextEdit... textEdits) {
```

Comment:
Again: is subclassing the ChangeConverter a use case?

Reply:
Again: I don't think so.